source/Outfit.cpp

```diff
@@ -189,12 +189,18 @@ void Outfit::Load(const DataNode &node)
 	if(node.Size() >= 2)
 	{
 		name = node.Token(1);
-		pluralName = name + 's';
+		displayName = name;
+		pluralName = displayName + 's';
 	}
 	isDefined = true;
 
 	for(const DataNode &child : node)
 	{
+		if(child.Token(0) == "display name" && child.Size() >= 2)
+		{
+			displayName = child.Token(1);
+			pluralName = displayName + 's';
```

Comment:
This will replace the pluralName if a pluralName was set by `plural` and then a `"display name"` is provided. You should instead remove this line, remove the same line above, then below this `for(const DataNode &child : node)` loop add the following:

```
if(pluralName.empty())
	pluralName = displayName + 's';
```

Reply:
done